extensions/functions_geometry.yaml

```diff
@@ -0,0 +1,39 @@
+%YAML 1.2
+---
+scalar_functions:
+  -
+    name: "st_point"
+    description: >
+      Return a point with the given x and y coordinate values.
+    impls:
+      - args:
+          - name: x
+            value: fp32
+          - name: y
+            value: fp32
+        return: point
+  -
+    name: "st_makeline"
```

Comment:
why is the point creation function name st_point but this function is st_makeline?

Reply:
Could we correct the naming here to avoid perpetuating their mistake?  st_makeline seems to be the problematic one as it's actually performing a join on two arbitrary geometries instead of two points.  Or is it making a line between the two geometries?

Reply:
The ST_ (Spatial Type) prefix isn't really necessary in Substrait as we're already associating all of the geometry functions within the geometry YAML file.  I believe we should probably use MakePoint and MakeLine to be compatible with what PostGIS supports.  Some Substrait implementations will only support the 2 argument version of MakePoint which would be the same as Point.